examples/drivers/windows/otApi/otApi.cpp

```diff
@@ -2168,7 +2168,7 @@ OTCALL
 otSendActiveSet(
     _In_ otInstance *aInstance, 
     const otOperationalDataset *aDataset, 
-    const uint8_t *aTlvs,
+    uint8_t *aTlvs,
```

Comment:
Keeping the `const` modifier seems to work for me.  You can see my [branch](https://github.com/openthread/openthread/compare/master...jwhui:xiaom-GitHub-pr/replace_session_id_tlv) and the build results ([travis](https://travis-ci.org/jwhui/openthread/builds/178043442) and [appveyor](https://ci.appveyor.com/project/jwhui/openthread-owfg0/build/0.1.577)).